### PR TITLE
changes for v1.1 release

### PR DESCRIPTION
**Pull request by rsnancollas**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -333,15 +376,47 @@ Source: [register.json](https:/github.com/edatasheets/digital-datasheets/blob/ma
 |bitFieldResetValue|Reset value of a bit field|String| |
 |bitFieldAccessType|Access type of a bit field|String| |
 
-####  4.5.22	 Specification Of Value
+####  4.5.25	 Description Of The Reliability Of a Component
```
Comment (asadate) [inline README.md]: "Specification" instead of "Description" to be consistent.

File: README.md [hunk 2]
```diff
@@ -71,21 +74,26 @@ A digital datasheet shall follow the international standard notation, YYYY-MM-DD
 
 ####  4.4	 Top Level Component Specification 
 
-Source: [component.json](https://github.com/edatasheets/edatasheets.github.io/blob/main/part-spec/component.json)
+Source: [component.json](https://github.com/edatasheets/digital-datasheets/blob/main/part-spec/component.json)
```
Comment (asadate) [inline README.md]: Change path to "github.io" throughout the document

File: README.md [hunk 3]
```diff
@@ -229,33 +235,17 @@ Source: [pinPaths.json](https://github.com/edatasheets/edatasheets.github.io/blo
 
 ####  4.5.14	 Specification Of Pins
 
-Source: [pinSpec.json](https://github.com/edatasheets/edatasheets.github.io/blob/main/part-spec/common/pinSpec.json)
+Source: [pinSpec.json](https://github.com/edatasheets/digital-datasheets/blob/main/part-spec/common/pinSpec.json)
```
Comment (asadate) [inline README.md]: It looks like "functionProperties" object definition is missing. Please check

**Review by asadate — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (rsnancollas) [reply README.md]: It's there, it just didn't change

File: README.md (hunk 1 above)
Comment (rsnancollas) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (rsnancollas) [reply README.md]: done and fixed the script

**Review by asadate — APPROVED** [review]

**Pull request merged**